CarAccidentErrorHandling/CarAccidentErrorHandling/Car.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace CarAccidentErrorHandling
+{
+    public class Car
+    {
+        public void Start()
+        {
+            Console.WriteLine("Car started.");
+        }
+
+        public void Run()
+        {
+            Console.WriteLine("Car is running.");            
+            DrivingScenario scenario = (DrivingScenario)new Random().Next(1, 6); // Simulate random car failure/accident scenarios
+            switch (scenario)
```

Comment:
Can it be done without switch case as it breaks OCP.

Reply:
I have used switch case just so I can simulate different accident scenarios randomly. In actual case, we would have a method inside the Run method which will throw any of the exceptions that we'll get.
Let me know what you think about it.

Reply:
Then it is ok else everything looks good to me.